usrtools/txgen/app/cmds.c

```diff
@@ -977,3 +983,168 @@ enable_capture(port_info_t *info, uint32_t state)
 {
     txgen_set_capture(info, state);
 }
+
+/**
+ *
+ * start_stop_latency_sampler - Starts or stops latency sampler.
+ *
+ * DESCRIPTION
+ * Starts or stops latency sampler.
+ *
+ * RETURNS: N/A
+ *
+ * SEE ALSO:
+ */
+
+void
+start_stop_latency_sampler(port_info_t *info, uint32_t state)
+{
+    if (state == ENABLE_STATE)
+        txgen_start_latency_sampler(info);
+    else if (state == DISABLE_STATE)
+        txgen_stop_latency_sampler(info);
+}
+
+/**
+ *
+ * txgen_start_latency_sampler - Starts latency sampler.
```

Comment:
The comments for this API and start_stop_latency_sampler are same, should they be?

Reply:
I see: 
```
start_stop_latency_sampler - Starts or stops latency sampler.
```
while this is

```
txgen_start_latency_sampler - Starts latency sampler.
```